src/main/java/com/epam/test_generator/api/BddGeneratorApi.java

```diff
@@ -140,7 +140,7 @@
 	)
 	@ContentType(JSON)
 	@POST("/projects")
-	static RestMethod createProject;
+	public static RestMethod createProject;
```

Comment:
If we need these fields `public` let's also make this change in `JdiHttpApiClientGenerator.java`

Reply:
added modifier